src/hotspot/share/gc/shenandoah/shenandoahVerifier.hpp

```diff
@@ -62,17 +62,17 @@ class ShenandoahVerifier : public CHeapObj<mtGC> {
     // Disable remembered set verification.
     _verify_remembered_disable,
 
-    // Assure old objects are registered and remembered set cards within the read-only remembered set are dirty
-    // for every interesting pointer within each OLD ShenandoahHeapRegion between bottom() and top().  This is
-    // appropriate at the init_mark safepoint since all TLABS are retired before we reach this code.
+    // Old objects should be registered and RS cards within *read-only* RS are dirty for all
+    // interesting pointers.
     _verify_remembered_for_marking,
```

Comment:
Right, did the renames to "before". Also moved the verification code from `ShenandoahHeap` to `ShenandoahVerifer`, since it logically belongs there.

But even the "before" thing deviates from the intent for `_verify_*` enums. They describe the _property_ we want to check, rather than the binding to a concrete phase. So it should probably be something like `_verify_rset_readonly` / `_verify_rset_readwrite_marked`? I'll leave that for future cleanup, along with cleanups in rset verification code.